src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.ProgressAdapter.cs

```diff
@@ -82,16 +82,16 @@ public IStreamingProgressTracker ProgressTracker
 
             // Do nothing functions.  The streaming far service doesn't care about
             // any of these.
-            public Task OnStartedAsync() => Task.CompletedTask;
-            public Task OnCompletedAsync() => Task.CompletedTask;
-            public Task OnFindInDocumentStartedAsync(Document document) => Task.CompletedTask;
-            public Task OnFindInDocumentCompletedAsync(Document document) => Task.CompletedTask;
+            public ValueTask OnStartedAsync() => default;
+            public ValueTask OnCompletedAsync() => default;
```

Comment:
just so i understand, but a `default` ValueTask` is one in the RanToCompletion state?

Reply:
i wish there was a ValueTask.CompletedTask, to at least be much clearer.

Reply:
Yeah, there is ValueTask.FromResult/CompletedTask but not on net472 :(

Reply:
Maybe we could define ValueTaskFactory class in Roslyn that has these.

Reply:
i would personally like that.  can come later though.

Reply:
OK, I'll follow up.